tests/unit/connection_tests.py

```diff
@@ -504,6 +483,47 @@ def test_close_channels(self):
         with self.assertRaises(AssertionError):
             self.connection._close_channels(200, 'reply text')
 
+    @mock.patch('pika.frame.ProtocolHeader')
```

Comment:
These tests were renamed and fixed up to call `Connection._on_connect_timer` instead of `Connection.connect`.